service/species-service.js

```diff
@@ -0,0 +1,28 @@
+
+const axios = require('axios')
+const log = global.log
+const ADMIN_BASE_API_URL = `${process.env.ADMIN_BASE_API_URL}`
+const SERVICE_NAME = 'species-service'
+
+async function createSpecies (payload) {
+  try {
+    let url = ADMIN_BASE_API_URL + '/species/product'
```

Comment:
/species/product?

Reply:
Yep, the species is associated with a product. I've gone this route for now as I suspect we will eventually be able to create a species disconnected from a product.